inlanecruising_plugin/include/inlanecruising_plugin/inlanecruising_plugin_node.hpp

```diff
@@ -0,0 +1,85 @@
+#pragma once
+
+/*
+ * Copyright (C) 2022 LEIDOS.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+#include <carma_planning_msgs/msg/plugin.hpp>
+#include <carma_ros2_utils/carma_lifecycle_node.hpp>
+#include <carma_guidance_plugins/tactical_plugin.hpp>
+#include <carma_wm_ros2/WMListener.hpp>
+#include <functional>
+#include "inlanecruising_plugin.hpp"
+#include "inlanecruising_config.hpp"
+#include <carma_planning_msgs/msg/trajectory_plan.hpp>
+#include <carma_planning_msgs/msg/trajectory_plan_point.hpp>
+#include <carma_planning_msgs/msg/plugin.hpp>
+#include <carma_planning_msgs/srv/plan_trajectory.hpp>
+#include <carma_debug_ros2_msgs/msg/trajectory_curvature_speeds.hpp>
+
+namespace inlanecruising_plugin
+{
+/**
+ * \brief ROS node for the InLaneCruisingPlugin
+ */ 
+class InLaneCruisingPluginNode : public carma_guidance_plugins::TacticalPlugin
+{
+public:
+    
+  /**
+   * \brief Node constructor 
+   */
+  explicit InLaneCruisingPluginNode(const rclcpp::NodeOptions &);
+
+  ////
+  // Overrides
+  ////
+  carma_ros2_utils::CallbackReturn on_configure_plugin();
+   
+  bool get_availability() override;
+
+  std::string get_plugin_name();
```

Comment:
This function is no longer needed after some changes on develop. You can remove. Runtime node name is used instead